site/docs/components/progress-bar.mdx

```diff
@@ -10,12 +10,13 @@ needToKnow:
 health:
   designed: true
   documented: true
-  implemented: false
-  latestDesign: false
-  allVariants: false
+  implemented: true
+  latestDesign: true
+  allVariants: true
   responsive: false
```

Comment:
I don't think anything would need to be done to make it responsive (it's naturally responsive in terms of width), but I'm also not sure if smaller screens call for a different design (such as font sizes), so I've left it as false.